catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/event/impl/SubscriptionImpl.java

```diff
@@ -22,13 +22,15 @@
 import ddf.catalog.event.Subscription;
 
 public class SubscriptionImpl implements Subscription {
-    private Filter filter;
+    protected Filter filter;
```

Comment:
Do we really need to make all the fields `protected` since most of them are initialized in the constructor and have a getter? For those that don't have getters, maybe we should consider adding on or making them `final`?